src/tty.c

```diff
@@ -60,3 +60,38 @@ static int luv_tty_get_winsize(lua_State* L) {
   lua_pushinteger(L, height);
   return 2;
 }
+
+#if LUV_UV_VERSION_GEQ(1, 33, 0)
+int luv_tty_set_vterm_state(lua_State* L)
+{
+  uv_tty_vtermstate_t state[] = {UV_TTY_SUPPORTED, UV_TTY_UNSUPPORTED};
+  const char* option[] = {"supported", "unsupported", NULL};
+  int idx = luaL_checkoption (L, 1, NULL, option);
+  uv_tty_set_vterm_state(state[idx]);
+  return 0;
+}
+
+int luv_tty_get_vterm_state(lua_State* L)
+{
+  uv_tty_vtermstate_t state;
+  int ret = uv_tty_get_vterm_state(&state);
+  if (ret < 0) return luv_error(L, ret);
+  switch (state)
+  {
+    case UV_TTY_SUPPORTED:
+      lua_pushliteral(L, "supported");
+      ret = 1;
+      break;
+    case UV_TTY_UNSUPPORTED:
+      lua_pushliteral(L, "unsupported");
+      ret = 1;
+      break;
+    default:
+      lua_pushinteger(L, state);
```

Comment:
It might be better to throw an error here? Maybe something like: `luaL_error(L, "unexpected uv_tty_vtermstate: %d", state);`